src/zzz_od/application/hollow_zero/lost_void/operation/lost_void_run_level.py

```
@@ -395,13 +513,17 @@ def try_interact(self) -> OperationRoundResult:
             current_interact_target = None
-            for ocr_result in ocr_result_map.keys():
+            for ocr_result in ocr_result_map:
                 target = match_interact_target(self.ctx, ocr_result)
                 if target is not None:
                     current_interact_target = target
                     break
 
             if current_interact_target is not None:
+                target_key = self.get_interact_target_key(current_interact_target)
+                if target_key in self.interacted_target_key_list:
+                    log.info('当前层已交互过 %s，本次不再交互，返回上游继续处理', target_key)
```

Ruff (0.15.18)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)